src/actions/index.js

```diff
@@ -1,6 +1,12 @@
-export const ACTION_EXAMPLE = "ACTION_EXAMPLE";
+export const SET_MENU_OPENED = "SET_MENU_OPENED";
+export const TOGGLE_MENU_OPENED = "TOGGLE_MENU_OPENED";
 
-export const actionExample = text => ({
-  type: ACTION_EXAMPLE,
+export const setMenuOpened = (state) => ({
+  type: SET_MENU_OPENED,
+  payload: state
+});
+
+export const toggleMenuOpened = () => ({
+  type: TOGGLE_MENU_OPENED,
   payload: {}
 });
```

Comment:
I think we can use only one of those actions.

Reply:
How would you go about it?